env/VERCEL.json

```diff
@@ -14,8 +14,8 @@
     "littera-core-ui": {
         "local_runner":  "self-hosted-runner-standard",
         "testing_workflow": "sharded_test",
-        "package_manager": "yarn",
-        "node_version": "16"
+        "package_manager": "berry",
```

Comment:
i think this needs to be a breaking change in core-ui, as in this repo should be versioned to (v9) because we will still be releasing in `yarn` for a brief period.

Reply:
resolving ansered in google meet. it will be Atomic responsibility to freeze anything and do the cutover basically no going back for core-ui.